md/web-service-tutorial.md

```diff
@@ -1,145 +1,114 @@
+
+11. Add a _**WebService SOAP 1.2**_ connector to this task and configure it using the wizard.
+
+12. In the **Connection parameters** window:
+* For the Community edition, provide these settings:
+  * Service NS: CountryInfoService
+  * Service name: http://www.oorsprong.org/websamples.countryinfo
+* For the Teamwork, Efficiency, or Performance edition. provide these settings:
+  * Enter the WDSL URL http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL, and click on the Introspect button, then leave the login info fields empty
+
+13. In the **Request parameters** window (for all editions), provide these settings:
+* Port name: CountryInfoServiceSoap12
+* End point address: http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso
+* Binding: http://www.w3.org/2003/05/soap/bindings/HTTP/
+* Envelope 
+```xml
+       <?xml version="1.0" encoding="utf-8"?>
+    <soap12:Envelope xmlns:soap12="http://www.w3.org/2003/05/soap-envelope">
+      <soap12:Body>
+        <CapitalCity xmlns="http://www.oorsprong.org/websamples.countryinfo">
+          <sCountryISOCode>${country}</sCountryISOCode>
+        </CapitalCity>
+      </soap12:Body>
+    </soap12:Envelope>
+```
+14. In the **Response configuration** window, check the **Returns body** checkbox to use the response body in the output operations.
+15. In the **Output operations** window, keep only one output operation:
+* **Left operand**: countryBo
+* **Operation**: Java method -> setCapital
+* **Right operand**: A script _parseCapital_  returning a String, with the following content
+ ```groovy
+import org.codehaus.groovy.ast.stmt.ContinueStatement
+import org.w3c.dom.NodeList
+
+responseDocumentBody.childNodes.item(0).textContent
+```
+
+And we are done for the connector configuration. If you want to test it from the wizard, replace _${country}_ by _FRA_ in the envelope, and ensure that _Paris_ is returned.
+
+16. Select the _**retrieveUnknownCapital**_ task, go on the _**Execution tab**_, click on _**Operations**_ and add the following operation:
+* **Left operand**: countryBo
+* **Operation**: Java method -> setId
+* **Right operand**: country
+
+17. Select the _**DisplayCapital**_ task, go on the _**Execution tab**_, click on _**Form**_ and click on the pencil to create the form of this task. The UI Designer should open. The idea here is to simply display the field _capital_ of the business object used in the case (which has been created during the case or retrieved from the database). This business object is accessible in the context.
+* Create a _**Javascript expression**_ variable named _**api**_, with the following content: 
+```Javascript
+return "../" + $data.context.countryBo_ref.link;
+```
+* Create an _**External API**_ variable named _**country**_, with the following api url:
+```
+{{api}}
+```
+* Insert a text widget in the form, with the following content: 
+```
+Capital: {{country.capital}}
+```
+
+Save the form, and we are done.  
```

Comment:
This is not blocking but the overview form has not been set for the pool so there is a warning on the diagram.